packages/xdl/src/logs/PackagerLogsStream.js

```diff
@@ -177,14 +177,21 @@ export default class PackagerLogsStream {
     ProjectUtils.attachLoggerStream(this._projectRoot, {
       stream: {
         write: chunk => {
-          if (chunk.tag !== 'metro' && chunk.tag !== 'expo') {
-            return;
-          } else if (this._getCurrentOpenProjectId() !== projectId) {
+          if (this._getCurrentOpenProjectId() !== projectId) {
             // TODO: We should be confident that we are properly unsubscribing
             // from the stream rather than doing a defensive check like this.
             return;
           }
 
+          if (chunk.tag === 'webpack') {
```

Comment:
This change can be removed after rebasing the branch, because we no longer call `_formatMsg` on the logs.

Reply:
This just makes nothing log out